src/main/java/Duke.java

```diff
@@ -1,10 +1,69 @@
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
 public class Duke {
-    public static void main(String[] args) {
+    public static void main(String[] args) throws IOException {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        String border = "____________________________________________________________\n";
+        System.out.print(border + " Hello! I'm Duke\n" + " What can I do for you?\n" + border);
+        String command;
```

Comment:
Maybe this need not be declared here, since you don't use it outside of the main while loop?

Reply:
Perhaps "command" is not the best name for this variable, since in your main while loop command is assigned to the whole line from `read.readLine()`; would it be better to name it something like `userInput` or `userInputLine` instead?